youtube_dl/extractor/sverigesradio.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    try_get,
+)
+
+
+class SverigesRadioBaseIE(InfoExtractor):
+    _GEO_COUNTRIES = ['SE']
+
+    def _extract_audio(self, audio_type, audio_id):
+        audiourl = self._download_json('https://sverigesradio.se/sida/playerajax/getaudiourl?type=' + audio_type + '&quality=high&format=iis&id=' + audio_id, audio_id)
+        metadata = self._download_json('https://sverigesradio.se/sida/playerajax/audiometadata?type=' + audio_type + '&id=' + audio_id, audio_id)
+        item = metadata['items'][0]
+
+        return {
+            'id': compat_str(audio_id),
+            'url': compat_str(audiourl['audioUrl']),
+            'title': compat_str(item['subtitle']),
+            'series': try_get(item, lambda x: x['title'], compat_str),
+            'duration': item.get('duration'),
+            'thumbnail': try_get(item, lambda x: x['displayimageurl'], compat_str),
+            'description': try_get(item, lambda x: x['description'], compat_str)
+        }
+
+
+class SverigesRadioArtikelIE(SverigesRadioBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?sverigesradio\.se/sida/artikel.aspx\?(.*)artikel=(?P<id>[0-9]*)'
```

Comment:
Do not capture empty strings.

Reply:
Fixed if you referring to `[0-9]*`? uses `+` now.
Now also don't capture things after `?`